README.md

```diff
@@ -307,6 +307,7 @@ _Source:_ [What is Docker](https://www.docker.com/why-docker)
 - [docker-bench-security](https://github.com/docker/docker-bench-security) - script that checks for dozens of common best-practices around deploying Docker containers in production. By [@docker][docker]
 - [docker-explorer](https://github.com/google/docker-explorer) - A tool to help forensicate offline docker acquisitions by [@Google][google]
 - [docker-lock](https://github.com/safe-waters/docker-lock) - A cli-plugin for docker to automatically manage image digests by tracking them in a separate Lockfile. By [@safe-waters][safe-waters]
+- [Checkov](https://github.com/bridgecrewio/checkov) - Static analysis for infrastructure as code manifests (Terraform, Kubernetes, Cloudformation, Helm, Dockerfile, Kustomize) find security misconfiguration and fix them. By [Bridgecrew](https://github.com/bridgecrewio)
```

Comment:
Hi @schosterbarak thanks for your contribution. Could you please insert 'Checkov' in the right alphabetical order (before 'Clair') and fix `[Bridgecrew]` to `[@Bridgecrew]`.

Thanks.